Wrappers.h

```diff
@@ -1,9 +1,14 @@
+#pragma once
 #include "BoulderProblem.h" //need to include because of nested class
 #include "TopRopeRoute.h"
-struct Mountain;
+
+// need to include because class owns instances of other objects
+#include "Mountain.h"  
+#include "TripPlan.h"
+#include "ClimbPlan.h"
```

Comment:
But you aren't USING any of these in Wrappers.h
you're using them in Wrappers.cpp
- [ ] forward declare in Wrappers.h
- [ ] `#include` in Wrappers.cpp